src/.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <!-- Required elements-->
+        <id>SwedbankPay.Sdk</id>
+        <description>A client library and SDK for Swedbank Pay's RESTful API Platform</description>
+        <authors>Swedbank Pay</authors>
+        <version>1.1.1</version>
+
+        <!-- Optional elements -->
+        <projectUrl>https://github.com/SwedbankPay/swedbank-pay-sdk-dotnet</projectUrl>
+        <license type="expression">Apache-2.0</license>
+        <icon>icon.png</icon>
+        <copyright>Swedbank Pay</copyright>
+        <tags>SwedbankPay sdk payments payment client api rest restful</tags>
+        <language>en-US</language>
+    </metadata>
+    <files>
```

Comment:
Shouldn't any of the project files or DDLs be referenced by the `.nuspec` file?

Reply:
I did not find any such parameter to pass inn. And the project file has to be provided when running `dotnet restore`

Reply:
As long as the published NuGet works, it's fine. Can we add a `nuget publish` and `nuget install` step in GitHub Actions so we know it all works as it should?

Reply:
I'll get on that